### PR TITLE
Exec Feb 22, 2023

### PR DESCRIPTION
**Pull request by astronautthis**

### CODE REVIEW

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: Template - [Executive Vote] Aave v2 D3M Deployment, Strategic Finance MKR Transfer, and MOMC Parameter Changes - February 22, 2023
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:2]:
```suggestion
title: Template - [Executive Vote] Aave v2 D3M Deployment, Strategic Finance MKR Vesting Streams, and MOMC Parameter Changes - February 22, 2023
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+summary: Onboards the Aave v2 Dai Direct Deposit Module, Strategic Finance Core Unit (SF-001) MKR Vesting Transfer, and MOMC parameter changes.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:3]:
```suggestion
summary: Onboards the new Aave v2 Dai Direct Deposit Module, Strategic Finance Core Unit (SF-001) MKR Vesting Stream setup, and MOMC parameter changes.
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+date: 2023-02-22T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Aave v2 D3M Deployment, Strategic Finance MKR Transfer, and MOMC Parameter Changes - February 22, 2023
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:8]:
```suggestion
# [Executive Proposal] Aave v2 D3M Deployment, Strategic Finance MKR Vesting Streams, and MOMC Parameter Changes - February 22, 2023
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- A total of **435 MKR** will be transferred to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP36), as detailed below.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:19]:
```suggestion
- A total of **435 MKR** will be streamed to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP36), as detailed below.
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+As per this [poll](https://vote.makerdao.com/polling/QmUMyywc#poll-detail), the Aave v2 D3M will be onboarded with the following parameters, if this executive proposal passes. 
+
+#### Vault Parameters
+
+* [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): **5 million DAI**.
+* [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **5 million DAI**.
+* [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **12 hours**.
+* [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar): **2%**.
+
+Note that Governance has approved a `line` of 20 million DAI but the intial deployment sets the `line` to 5 million DAI following the advice of the Protocol Engineering Core Unit.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:48]: Bold for emphasis

```suggestion
**Note that Governance has approved a `line` of 20 million DAI but the intial deployment sets the `line` to 5 million DAI following the advice of the Protocol Engineering Core Unit**. 
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+#### Vault Parameters
+
+* [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line): **5 million DAI**.
+* [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap): **5 million DAI**.
+* [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl): **12 hours**.
+* [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar): **2%**.
+
+Note that Governance has approved a `line` of 20 million DAI but the intial deployment sets the `line` to 5 million DAI following the advice of the Protocol Engineering Core Unit. 
+
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:49]:
```suggestion

For more information on the DAI Direct Deposit Module, see the entry in the Maker Operational Manual [here](https://manual.makerdao.com/module-index/module-dai-direct-deposit).

```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+#### Threads and Assessments 
+
+Please review the following resources containing information about the Aave V2 D3M to inform your position before voting.
+* [Proposal Thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-2-february-2023/19637)
+* [Initial Risk Core Unit Evaluation](https://forum.makerdao.com/t/d3m-risk-assessment/9353)
+* [Initial Protocol Engineering Core Unit Evaluation](https://forum.makerdao.com/t/direct-aavev2-dai-direct-deposit-module-technical-assessment/10751)
+* [MIP 50: DAI Direct Deposit Module](https://mips.makerdao.com/mips/details/MIP50)
+
+
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:58]: remove empty line

```suggestion
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+### Strategic Finance Core Unit (SF-001) MKR Transfer
+
+As per their successful [MKR vesting proposal](https://vote.makerdao.com/polling/QmZn66qt), the following MKR streams will be added to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP36) starting March 1, 2022 through March 1, 2025 in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), if this executive passes. The MKR is claimable on March 1, 2023.
+- **240 MKR** will be streamed to [0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD](https://etherscan.io/address/0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD). 
+- **195 MKR** will be streamed to [0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9](https://etherscan.io/address/0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9)
+
+$>>> Should there be a comment that this is legacy work since the stream start date is in the past? <<< $
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:65]: agree here something like:

```suggestion
Note these stream and cliff dates have been backdated to the date that the contributors started work with the Strategic Finance Core Unit.
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+* [Initial Risk Core Unit Evaluation](https://forum.makerdao.com/t/d3m-risk-assessment/9353)
+* [Initial Protocol Engineering Core Unit Evaluation](https://forum.makerdao.com/t/direct-aavev2-dai-direct-deposit-module-technical-assessment/10751)
+* [MIP 50: DAI Direct Deposit Module](https://mips.makerdao.com/mips/details/MIP50)
+
+
+### Strategic Finance Core Unit (SF-001) MKR Transfer
+
+As per their successful [MKR vesting proposal](https://vote.makerdao.com/polling/QmZn66qt), the following MKR streams will be added to the [Strategic Finance Core Unit (SF-001)](https://mips.makerdao.com/mips/details/MIP39c2SP36) starting March 1, 2022 through March 1, 2025 in accordance with their active [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), if this executive passes. The MKR is claimable on March 1, 2023.
+- **240 MKR** will be streamed to [0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD](https://etherscan.io/address/0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD). 
+- **195 MKR** will be streamed to [0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9](https://etherscan.io/address/0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9)
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:63]: Improve readability

```suggestion
As per their successful [MKR vesting proposal](https://mips.makerdao.com/mips/details/MIP40c3SP48), the following MKR streams will be added for the [Strategic Finance Core Unit (SF-001)], if this executive proposal passes. 

#### Contributor Stream 1

- Amount to be Streamed: **240 MKR**
- Start Date: **2022-03-01**
- End Date: **2025-03-01**
- Cliff Date: **2023-03-01**
- Destination Address: [0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD](https://etherscan.io/address/0x31C01e90Edcf8602C1A18B2aE4e5A72D8DCE76bD)

#### Contributor Stream 2

- Amount to be Streamed: **195 MKR**
- Start Date: **2023-03-01**
- End Date: **2025-03-01**
- Cliff Date: **2023-03-01**
- Destination Address: **[0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9](https://etherscan.io/address/0x12b19C5857CF92AaE5e5e5ADc6350e25e4C902e9)**
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+* Reduce the RETH-A Stability Fee by 1% from 1.5% to **0.5%**.
+* Reduce the WBTC-A Stability Fee by 0.25% from 2% to **1.75%**.
+* Reduce the WBTC-B Stability Fee by 0.25% from 3.5% to **3.25%**.
+
+##### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) increases
+
+* Increase the CRVV1ETHSTETH-A Maximum Debt Ceiling (`line`) from 20 million DAI to **100 million DAI**.
+* Increase the RETH-A Maximum Debt Ceiling (`line`) from 5 million DAI to **10 million DAI**.
+* Increase the MATIC-A Maximum Debt Ceiling (`line`) from 10 million DAI to **15 million DAI**.
+* Increase the Compound v2 D3M Maximum Debt Ceiling (`line`) from 20 million DAI to **30 million DAI**.
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:85]: add deltas here

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:87]:
```suggestion
```

File: governance/votes/Executive Vote - February 22, 2023.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+---
+
+## Resources
+
+Additional information about the Governance process can be found in the [Governance](https://community-development.makerdao.com/en/learn/governance) section of the MakerDAO community portal.
+
+To participate in future Governance calls, please [join us](https://github.com/makerdao/community/tree/master/governance/governance-and-risk-meetings) every Thursday at 17:00 UTC.
+
+To add current and upcoming votes to your calendar, please see the [MakerDAO Public Events Calendar](https://calendar.google.com/calendar/embed?src=makerdao.com_3efhm2ghipksegl009ktniomdk%40group.calendar.google.com&ctz=UTC&mode=week&showCalendars=0&showPrint=0).
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - February 22, 2023.md:104]: fetch new template which updates the links here please - https://github.com/makerdao/community/blob/master/governance/votes/templates/Executive%20vote%20-%20%24date_MONTH_DD%2C_YYYY.md

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Good start - please see comments

File: governance/votes/Executive Vote - February 22, 2023.md (hunk 9 above)
Comment (astronautthis) [reply governance/votes/Executive Vote - February 22, 2023.md]: Just double checking that this isn't a typo: Contributor 2's stream start date is also 2022 in the spreadsheet

Comment (astronautthis) [reply governance/votes/Executive Vote - February 22, 2023.md]: Also, wasn't sure if you meant to delete the link in this part "MKR streams will be added for the [Strategic Finance Core Unit (SF-001)], if this executive proposal passes." or meant for me to add the link. I'll add it, but feel free to remove if you think that's better.

Comment (patrick-j-govalpha) [reply governance/votes/Executive Vote - February 22, 2023.md]: > Just double checking that this isn't a typo: Contributor 2's stream start date is also 2022 in the spreadsheet

This was a typo - ARGH

**Review by patrick-j-govalpha — APPROVED** [review]
LGTM - thanks!

**Pull request merged**